tests/table/test_eval.py

```diff
@@ -263,6 +271,19 @@ def test_table_numeric_trigonometry(fun: str) -> None:
     )
 
 
+def test_table_numeric_cot() -> None:
```

Comment:
This can not merge into `test_table_numeric_trigonometry` as the cotangent is not available in numpy.

Reply:
makes sense!